cv-data-service-library/src/main/java/com/trihydro/library/helpers/TimGenerationHelper.java

```diff
@@ -1079,19 +1055,19 @@ private String updateAndSendRSU(WydotTravelerInputData timToSend, TimUpdateModel
         } else {
             // we don't have any existing RSUs, but some fall within the boundary so send
             // new ones there. We need to update requestName in this case
-            for (int i = 0; i < dbRsus.size(); i++) {
-                String regionName = getRsuRegionName(aTim, dbRsus.get(i));
+            for (WydotRsu rsu : dbRsus) {
+                String regionName = getRsuRegionName(aTim, rsu);
                 try {
                     regionName = regionNameTrimmer.trimRegionNameIfTooLong(regionName);
                 } catch (IllegalArgumentException e) {
-                    utility.logWithDate("Failed to trim region name: " + e.getMessage());
+                    log.error("Failed to trim region name", e);
```

Comment:
suggestion(non-blocking): use e.getMessage() to reduce noise in the logs. Using `log.error(..., exception)` will prompt the logger to write the entire exception stacktrace to the logs. This can be noisy and unhelpful, but sometimes it can be incredibly helpful. Since this exception will occur during a "trim string" operation, there won't be much value to the stacktrace.

```suggestion
                    log.error("Failed to trim region name: {}", e.getMessage());
```

Reply:
Makes sense, addressed!